Appraisals

```diff
@@ -1482,10 +1482,10 @@ elsif ruby_version?('3.0') || ruby_version?('3.1')
   (3..5).each { |v| gem_cucumber(v) }
 
   appraise 'contrib' do
-    gem 'actionpack'
-    gem 'actionview'
+    gem 'actionpack', '~> 7'
```

Comment:
The first time I bundled the gems, they actually being downgraded from 7 to 6. In order to be consistent, I added `~> 7` to keep the dependency at 7. I have removed the `~> 7` and bundled again, and it stays at `7`. 

I would avoid trying update to 8 now, since that would not be relevant to this PR.